### ISSUE TITLE
[Angular] Hooking into close function

### ISSUE THREAD

**Issue by darkmantle** [manijak/nativescript-photoviewer#42]
Is it possible to hook into the event when the gallery is closed? I have a timer I need to stop/start depending on the gallery being in focus.

**Comment by darkmantle** [comment]
Ok, just seen the demo, sorry!

**Comment by manijak** [comment]
It is, but I think it’s a bit faulty on the android side

**Comment by darkmantle** [comment]
It's not working at all on iOS. I have this:

```ts
openGallery() {
        this.endTimer();
        const photoOptions: PhotoViewerOptions = {
            startIndex: this.currentImage,
            ios: {
                completionCallback: this.galleryLoaded
            }
        }

        const photoView = new PhotoViewer();
        photoView.showGallery(this.photos).then(() => {
            console.log("gallery closed");
            this.startTimer();
        });
    }
```

**Comment by manijak** [comment]
Should be fixed in the latest version (2.1.0)

**Comment by darkmantle** [comment]
Yes, works great now. Cheers.
